bin/run_docker.py

```diff
@@ -42,6 +42,37 @@ def get_submission_image(syn: synapseclient.Synapse, submission_id: str) -> str:
     return image_id
 
 
+def truncate_log_file(log_file_name: str,
+                      log_file_path: str,
+                      max_lines: int = 20) -> None:
```

Comment:
IMO This is much too low of a number to log - My feeling is to truncate based on size, but I suppose line count also works.

If you keep line count I would default to a number in the hundreds.

Reply:
++ here.  I think historically, we had some functionality to only return the last 5 lines of a log file. BUT, it may be worthwhile to ask CNB what they think since we are implementing this again.

Reply:
@vpchung looping you in to discuss the upper-limit for the number of lines captured in the Docker execution log for model-to-data challenges! I changed it to get the last 20 lines by default, but what do you think of bumping this up to 150? based on your past experience with previous model submissions.

Reply:
> historically, we had some functionality to only return the last 5 lines of a log file

Yes, but to clarify, we only returned the last 5 lines IF the file size is over 50Kb.  If it's less, then we'll upload the entire log.

Reply:
Based off conversation with CNB (thanks @vpchung!) I'm going to move forward with the following:

1. Default line size will be 20 lines: Rong brought up that too much information (too many lines) shared with participants can open up the door for cheating when fine-tuning their models.
2. A file size cap will be used instead of the line # cap. Tom mentioned the possibility of bypassing the line limit by condensing all the stdout info into a small number of lines (he claims this is not from personal experience 👀 )

Thanks all